hudi-common/src/main/java/org/apache/hudi/metadata/MetadataPartitionType.java

```diff
@@ -137,6 +148,10 @@ public String getFileIdPrefix() {
     return fileIdPrefix;
   }
 
+  public int getRecordType(String key) {
+    return recordType;
```

Comment:
The key is never used?

Reply:
only used for FILES because FILES partition has two different record types. I have added another method without the key argument as suggested by @bvaradar so that we don't have to pass empty string all the time.